hw8/task1.py

```diff
@@ -0,0 +1,51 @@
+from collections import defaultdict
+
+
+class KeyValueStorage:
+    """
+        This is a wrapper class  for some key-value storage file.
+         File consists rows,  each other is represented as key and value separated by = symbol.
+
+         This class has its(storage) keys and values accessible as collection items and as attributes.
+    """
+
+    def __init__(self, file_path):
+        """
+        :param file_path: type: str, takes path to file which is a key-value storage file.
```

Comment:
type: str - what is the style of annotation of type?

Reply:
Fixed